lib/yang.c

```diff
@@ -607,6 +607,9 @@ static void ly_log_cb(LY_LOG_LEVEL level, const char *msg, const char *path)
 	case LY_LLVRB:
 		priority = LOG_DEBUG;
 		break;
+	case LY_LLDBG:
+		priority = LOG_DEBUG;
+		break;
 	default:
```

Comment:
Please remove the default.  It is not needed.  LY_LOG_LEVEL is a enum, let's treat it as such.

Reply:
👍 I removed it and initialized `priority` before the case statement, just to prevent compiler warnings.